lib/core/tree.js

```diff
@@ -149,8 +149,38 @@ class TernarySearchTree {
 
 const tree = new TernarySearchTree()
 
-for (let i = 0; i < wellknownHeaderNames.length; ++i) {
-  const key = headerNameLowerCasedRecord[wellknownHeaderNames[i]]
+// function shuffleArray (array) {
```

Comment:
The plan is to shuffle them upon first script evaluation or how exactly?

Reply:
My plan is to generate statically(before any script evaluation), and I include how to generate it in a file that will not be loaded. Is this OK?

Reply:
So, it will need to be generated before actually publishing the library?

I'm ok either way, I believe I find generating the array a single time upon script valuation a bit easier to maintain, tho don't have a strong preference.